content/en/docs/concepts/storage/volume-attributes-classes.md

```diff
@@ -0,0 +1,95 @@
+
+<!-- body -->
+
+## Introduction
+
+A VolumeAttributesClass provides a way for administrators to describe the mutable "classes" of storage they offer. Different classes might map to quality-of-service levels. Kubernetes itself is unopinionated about what classes represent. 
+
+
+## The VolumeAttributesClass Resource
+
+Each VolumeAttributesClass contains the `driverName` and `parameters`, which are used when a PersistentVolume belonging to the class needs to be dynamically provisioned or modified.
+
+The name of a VolumeAttributesClass object is significant, and is how users can request a particular class. Administrators set the name and other parameters of a class when first creating VolumeAttributesClass objects. While the name of a VolumeAttributesClass object in a PersistentVolumeClaim is mutable, the parameters in an existing class are immutable.
+
+
+```yaml
+apiVersion: storage.k8s.io/v1alpha1
+kind: VolumeAttributesClass
+metadata:
+  name: silver
+driverName: pd.csi.storage.gke.io
+parameters:
+  provisioned-iops: "3000"
+  provisioned-throughput: "50" 
+```
+
+
+### Provisioner
+
+Each VolumeAttributes has a provisioner that determines what volume plugin is used for provisioning PVs. This field must be specified. 
+
+The feature support for VolumeAttributesClass is implemented in [kubernetes-csi/external-provisioner](https://github.com/kubernetes-csi/external-provisioner).
+
+You are not restricted to specifying the [kubernetes-csi/external-provisioner](https://github.com/kubernetes-csi/external-provisioner). You can also run and specify external provisioners, which are independent programs that follow a specification defined by Kubernetes. Authors of external provisioners have full discretion over where their code lives, how the provisioner is shipped, how it needs to be run, what volume plugin it uses (including Flex), etc. 
+
+
+### Resizer
+
+Each VolumeAttributes has a resizer that determines what volume plugin is used for modifying PVs. This field must be specified. 
+
+The modifying volume feature support for VolumeAttributesClass is implemented in [kubernetes-csi/external-resizer](https://github.com/kubernetes-csi/external-resizer).
+
+For example the existing PVC is using VolumeAttributesClass silver:
+
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: test-pv-claim
+spec:
+  …
+  volumeAttributesClassName: gold
+  …
+```
+
+
+A new VolumeAttributesClass gold is available in the cluster:
+
+
+```yaml
+apiVersion: storage.k8s.io/v1alpha1
+kind: VolumeAttributesClass
+metadata:
+  name: gold
+driverName: pd.csi.storage.gke.io
+parameters:
+  iops: "4000"
+  throughput: "60"
+```
+
+
+The end user can update the PVC with the new VolumeAttributesClass gold and apply:
+
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: test-pv-claim
+spec:
+  …
+  volumeAttributesClassName: gold
+  …
+```
+
+
+## Parameters
```

Comment:
I think it's better if there's a description about the relation and difference between the VAC's parameters and the SC's parameters. 

for SC, the parameters fields can holds both immutable volume attributes and mutable volume attributes. 
for VAC, the parameter fields should hold mutable volume attributes and takes precedence over the SC's parameters fields if there's any conflict between them when creating a volume.

Reply:
for SC, the parameters fields can holds both immutable volume attributes and mutable volume attributes. This may not be accurate, we may want to encourage users to put mutable attributes in VAC instead of SC. 

We